packages/rs-sdk/src/platform/put.rs

```diff
@@ -0,0 +1,53 @@
+//! Module containing the Put trait, allowing modification of objects (state transitions) on the Dash Platform.
```

Comment:
this file is unused as state transitions turned out to be much more complex than I expected. To be removed.